loader/app/transform/util.py

```diff
@@ -39,49 +47,82 @@ def split_and_merge_urls(doc_urls, sep):
     return merged_doc_urls
 
 
-def get_urls(dataframe, policy_id=None, sep=';', sub_sep=None, url_sep=','):
-    country_code = dataframe['country_code']
+def get_policy_data(dataframe, sep=';', sub_sep=None, url_sep=',') -> Optional[Tuple[Key, PolicyData]]:
     policy_name = dataframe['policy_name']
+    country_code = dataframe['country_code']
+    policy_type = CCLWActionType[dataframe["policy_type"]].value
```

Comment:
I note that this mapping and the possible values for `Action Type` are hardcoded in code. If our definitions of Action Type change, we'll need to remember to update the mapping in `CCLWActionType`. Perhaps no obvious was round this, but just wanted to mention it.

Reply:
Yes, this is just hard-coded for alpha. Ideally, it would be managed via DB or something.